### DISCUSSION TITLE
Welcome to repo-1 Discussions!

### DISCUSSION THREAD

**Discussion by steeve-orbit** [sally-playground/repo-1#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by steeve-orbit** [comment]
First comment

**Reply by steeve-orbit** [reply]
First reply to first comment

**Reply by steeve-orbit** [reply]
Second reply to first comment

**Comment by steeve-orbit** [comment]
Second comment

**Comment by Bogda** [comment, marked as answer]
Hello there 🤖
